lib/internal/http2/core.js

```diff
@@ -355,7 +447,7 @@ class Http2Session extends EventEmitter {
 
   get destroyed() {
     const state = this[kState];
-    return Boolean(state.destroyed);
+    return !!state.destroyed;
```

Comment:
do we need this check at all? I think we should always have a `destroyed` property in the state.